docs/src/dev/provider/gremlin-semantics.asciidoc

```diff
@@ -1543,7 +1543,7 @@ Null values from the incoming traverser are not processed and remain as null whe
 * For `Scope.global` or parameterless function calls, if the incoming traverser is a non-String value then an `IllegalArgumentException` will be thrown.
 * For `Scope.local`, if the incoming traverser is not a string or a list of strings then an `IllegalArgumentException` will be thrown.
 
-See: link:https://github.com/apache/tinkerpop/tree/x.y.z/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/ToLowerStep.java[source],
+See: link:https://github.com/apache/tinkerpop/tree/x.y.z/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/ToLowerGlobalStep.java[source],
```

Comment:
We should probably add a second link for the local versions of all of these steps as well.

Reply:
Dedup only links to the global one, so I'm leaving it as the global one (since it's also the default)

Reply:
Hmm, I wrote the dedup semantics docs, I think that was an oversight on my part